lib/Sema/CSGen.cpp

```diff
@@ -3061,9 +3084,10 @@ namespace {
 
       for (auto pack : expandedPacks) {
         Type packType;
-        if (auto *elementExpr = getAsExpr<PackElementExpr>(pack)) {
-          packType = CS.getType(elementExpr->getPackRefExpr());
-        } else if (auto *elementType = getAsTypeRepr<PackElementTypeRepr>(pack)) {
+        if (isExpr<PackElementExpr>(pack)) {
+          continue;
```

Comment:
Nit: Could you please leave a comment here explaining why we skip over pack elements and where the connection between this pack expansion and pack element is established now to make it easier in the future to figure out what is going on here?